server/src/endpoints/endpoints.py

```diff
@@ -1,4 +1,8 @@
+import os
+import pathlib
+from flask import Flask, redirect
 from flask_restx import Resource
+from google_auth_oauthlib.flow import Flow
```

Comment:
this prob isn't needed here?

Reply:
and can you get rid of the hello word here as well haha tyty

Reply:
Oh yea mb. That import shouldn't be there